components/places/src/places.udl

```diff
@@ -5,7 +5,7 @@
 [Custom]
 typedef string Url;
 [Custom]
-typedef i64 Timestamp;
+typedef i64 PlacesTimestamp;
```

Comment:
This seemed like the least destructive way, this makes the change non-breaking

Normally I wouldn't mind a breaking change but the rename here is cheap and the breaking change in iOS is large (there is at least 30+ uses of `Timestamp` that would be ambigous) - if anyone feels strongly about it we can go the breaking route